### DISCUSSION TITLE
What a year of Milestones! 🏁 - Actions and Packages check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#42110] [Actions]
Hello everyone!👋  2022 has been an amazing year for GitHub Actions and Packages so far! We have come a long way with a series of hot and exciting Actions and Packages ships. This will be the last check-in for 2022 and so I'll keep it short. Let's go!

## 💌  From Community - Shoutout 📢  to these Self-Solvers!

Community members getting better at self-solving! ✅  These self-solves are proof of members hustling and self-troubleshooting problems to solutions. 

- [@robotgryphon](https://github.com/robotgryphon) was facing issues building mod repos but quickly realized the [root cause](https://github.com/orgs/community/discussions/41089) by digging deeper with debug and stacktrace flags. Great job 👏

- We all use Marketplace Actions and why shouldn’t we? They are super useful! And so is [this thread](https://github.com/orgs/community/discussions/40094) from [@allwalte](https://github.com/allwalte), where a simple workflow change gave multiple errors. Glad that this was resolved by [@allwalte](https://github.com/allwalte)  and we owe him a BIG 👍 for coming back and updating the discussion.

- Another amazing self-troubleshooting from [@ferferga](https://github.com/ferferga) in [his post](https://github.com/orgs/community/discussions/39408) where he meticulously recognized the property missing from one of the steps. Thank You! 💗

## From the Release Radar 🚨 

- [Fine-grained permissions and organization-level publishing are now available for the GitHub Packages NuGet registry](https://github.blog/changelog/2022-12-09-packages-fine-grained-permissions-and-organization-level-publishing-are-now-available-for-the-github-packages-nuget-registry/). 🔐
The GitHub Packages NuGet registry now runs on a new architecture, unlocking great new capabilities. [Read more about the change](https://github.blog/changelog/2022-12-09-packages-fine-grained-permissions-and-organization-level-publishing-are-now-available-for-the-github-packages-nuget-registry/).

- [GitHub Actions workflow notifications in Slack and Microsoft Teams](https://github.blog/changelog/2022-12-06-github-actions-workflow-notifications-in-slack-and-microsoft-teams/) 🔔
Another solid ship! 🚢  GitHub app in Slack and Microsoft Teams now supports GitHub Actions workflow notifications. You can now subscribe to your repository and get notified about GitHub Actions workflow run status from your channel or personal app. I found it very useful as it helps me stay on top of the notifications even when I am away from my computer. For more information, please visit the GitHub app guidance for [Slack](https://github.com/integrations/slack/blob/main/README.md#actions-workflow-notifications) and [Microsoft Teams](https://github.com/integrations/microsoft-teams/blob/main/Readme.md#actions-workflow-notifications).

## Make docs your true friend - [Docs are being translated into new languages](https://github.blog/changelog/2022-12-05-docs-are-being-translated-to-french-german-korean-and-russian/) 🔖 

In our commitment to helping you learn easily and natively, we’re translating [GitHub Docs](https://docs.github.com/) into four new languages – French, German, Korean, and Russian! These are in addition to the current languages – Brazilian Portuguese, Chinese, Japanese, and Spanish. From the top of any GitHub Docs page, click the dropdown to change the language you're viewing. 

Do you have any feedback around docs that needs any improvement or doesn’t cover everything you’d like to see around a specific feature? Feel free to share this with our Docs team. Alternatively, you can always [contribute](https://github.com/github/docs/edit/main/content/index.md) directly to any doc by submitting a PR–learn more about [how to contribute](https://github.com/github/docs/blob/main/CONTRIBUTING.md) to docs.

[@supergranular](https://github.com/supergranular) from the Docs team at GitHub is looking into [how you use docs](https://github.com/orgs/community/discussions/40648) alongside the GitHub products. Please fill out this [4-question survey 🗳️ ](https://github.surveymonkey.com/r/docs-issues-research) and help us gather your thoughts.

## Good Reads! 📚 

- Our community's [Discover Category](https://github.com/orgs/community/discussions/categories/discover) has been thriving and we have some great content from subject matter experts and industry leaders that you should not miss out on. Read this [masterpiece](https://github.com/orgs/community/discussions/41489) from [@ayodejiayodele](https://github.com/ayodejiayodele) where he has explained some of the common pitfalls that often plague DevOps teams or organizations.
- How much does it really cost to buy more powerful cloud computing resources for development work? A lot less than you think. To find out, [read this blog post](https://github.blog/2022-12-08-experiment-the-hidden-costs-of-waiting-on-slow-build-times/) by [@somenatalie](https://github.blog/author/somenatalie/) as the results may surprise 😯 you.
- GitHub completed 10 years of Octoverse this month. Today, we live in a very different world and a lot has changed over the years. Something that has not changed, is our commitment to Open Source Software. Discover all the latest key trends and see how open source is changing software development in 2022. [Check out the state of open source software(OSS) report for 2022](https://octoverse.github.com/).

Oh! Do you like GitHub-branded hoodies, joggers, stickers, and more? ⏰ 🛍️ There's a new sale 🛒  on the [GitHub Shop](https://thegithubshop.com/collections/sale-2022) 🛍️  that we don't think you'll want to miss out on. Get it before it's gone for the year! 

With that, it’s time to wrap up the last check-in for this year. I hope you enjoy the rest of the year 😃 and I can’t wait to see what Actions we’ll Package together and accomplish in 2023. 🚀

**Comment by ssbarnea** [comment]
Nice link to shop that gives DNS_PROBE_FINISHED_NXDOMAIN  on https://thegithubshop.com/collections/sale-2022

**Reply by Akash1134** [reply]
Hi @ssbarnea , the link takes me to the correct webpage. Could you please check the same on a different device or network? Thanks!
